### PR TITLE
New formula: Lacaille 2.2.2

### PR DESCRIPTION
**Pull request by umireon**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

---

[Lacaille] is a keyboard remapper for the Japanese thumb-shift method (親指シフト).

[Lacaille]: http://lacaille.jpn.org

### CODE REVIEW

File: Casks/lacaille.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'lacaille' do
+  version '2.2.2'
+  sha256 '3885a55a4fe1b80f9340271972372ab4b4fe234aad37c27f849e6d78fbd6fcb2'
+
+  url "http://lacaille.jpn.org/Lacaille_#{version}.dmg"
+  appcast 'http://lacaille.jpn.org',
```
Comment (vitorgalvao) [inline Casks/lacaille.rb]: Why do you think this is an effective appcast? I does not appear like this page will change with every version.

File: Casks/lacaille.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+          checkpoint: '9278d72ed9b5179467c1cf8888fc8d981ba32fc085bad32cf09119cf836c3ec4'
+  name 'Lacaille'
+  homepage 'http://lacaille.jpn.org'
+
+  app 'Lacaille.app'
+
+  uninstall login_item: 'Lacaille'
+
+  zap delete: '~/Library/Preferences/org.jpn.lacaille.Lacaille.plist'
```
Comment (vitorgalvao) [inline Casks/lacaille.rb]: Made this into a string (no point in an array for a single element).

File: Casks/lacaille.rb (hunk 1 above)
Comment (umireon) [inline Casks/lacaille.rb]: I'm sorry for that being unclear.
I suggested http://lacaille.jpn.org is an effective appcast for the following reason.

http://lacaille.jpn.org is the only page the site has.
In the second section of the page which means Downloads and New Features (ダウンロード・新機能), the list of Lacaille releases is presented.
Therefore, this page must change with every release.

**Comment by umireon** [comment]
@vitorgalvao Thank you for reviewing!
I think the appcast is valid, but unfortunately the releases is hidden until clicking the header.

**Comment by vitorgalvao** [comment]
No matter that they’re hidden, since they’re in the HTML. Took another look and it seems like it might work as an appcast, yes. Thank you.

**Pull request merged**